lib/galaxy/datatypes/converters/fastqcssangergz_to_fastqcssanger.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="CONVERTER_fastqcssangergz_to_fastqcssanger" name="Convert fastqcssanger.gz files to fastqcssanger" version="1.0.0" hidden="true">
+  <command>gzip -dcf $input1 > $output1</command>
```

Comment:
Hmm, even the busybox and ubuntu docker base images come with gzip, os x has a system gzip ... I doubt we could satisfy a dependency or run galaxy without gzip.
Alternatively I could write a python script to do the decompression, if that's a concern.